### ISSUE TITLE
Visual notifications

### ISSUE THREAD

**Issue by stevedekorte** [Voluntarynet/voluntary.app#7]
Need to add support for web visual notifications.

**Comment by ConProgramming** [comment]
Do you think this would overall work better, especially for mobile, as a PWA? They make desktop notifications really easy too

**Comment by stevedekorte** [comment]
I'm not sure I understand the question. Isn't the project currently a PWA? What does being a PWA have to do with browser notifications?

**Comment by stevedekorte** [comment]
Ah, do you mean the Android specific notifications associated with telling it to treat a web page as a mobile app? I'm open to supporting that as an option when available but we need some notifications to work across browsers and platforms as well.

**Comment by ConProgramming** [comment]
> Ah, do you mean the Android specific notifications associated with telling it to treat a web page as a mobile app? I'm open to supporting that as an option when available but we need some notifications to work across browsers and platforms as well.

No, I mean [this](https://developers.google.com/web/progressive-web-apps/desktop) they work on iOS, Android, Linux, Windows, and soon Mac to easily provide native notifications and some other stuff.

**Comment by stevedekorte** [comment]
Thanks - that sounds great. I'd love to support this.
